src/agent/firebase-controller.ts

```diff
@@ -184,32 +212,43 @@ export class FirebaseController implements Controller {
     // https://firebase.google.com/docs/reference/rest/database#section-server-values
     breakpoint_map['finalTimeUnixMsec'] = {'.sv': 'timestamp'};
 
-    this.db
-      .ref(`cdbg/breakpoints/${this.debuggeeId}/active/${breakpoint.id}`)
-      .remove();
+    try {
+      await this.db
+        .ref(`cdbg/breakpoints/${this.debuggeeId}/active/${breakpoint.id}`)
+        .remove();
```

Comment:
Updates get queued up and retried on failure outside of here right? Will if remove() has already been done successfully on one attempt, I'm assuming it won't 'fail' on a subsequent?

Reply:
I've confirmed that removing something that is not present will succeed.

Yes, updates are retried if they fail.